internal/bgp/native/native.go

```diff
@@ -98,9 +98,14 @@ func (sm *sessionManager) SyncExtraInfo(extras string) error {
 
 func (sm *sessionManager) SetEventCallback(func(interface{})) {}
 
+func (s *session) PeerLabel() string {
```

Comment:
no need for this to be a method. Let's have a `peer` (or let's find a better name) private field we set in `NewSession` above